taas/mapping.py

```diff
@@ -0,0 +1,99 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Mapping(object):
+    """
+        This class handles mappings between our raw input, and what we generate.
```

Comment:
I don't know Python... Is this a comment which describes the responsibilities of the Class and that we can pull into documentation somewhere?